### PR TITLE
Fixup class docstrings

### PR DESCRIPTION
**Pull request by schaefi**
The attribute list should provide information about the
construction of an object of this class. Some fields
were missing or superfluous

### CODE REVIEW

**Comment by schaefi** [comment]
@davidcassany would you mind to take a short look at this one. I stumbled over this when implementing the xz compression options and thought this should be cleaned up first.

Thanks

**Comment by davidcassany** [comment]
@schaefi sure I am having a look

File: kiwi/builder/disk.py [hunk 1]
```diff
@@ -61,115 +61,17 @@ class DiskBuilder(object):
...
 
-    * :attr:`generic_fstab_entries`
-        List of generic/persistent fstab entries
+    * :attr:`root_dir`
+        Root directory path name
 
-    * :attr:`result`
-        Instance of Result
+    * :attr:`custom_args`
+        Custom processing arguments defined as hash keys:
```
Comment (davidcassany) [inline kiwi/builder/disk.py]: nit-pick, but I guess the colons should not be there if no key-values defined for the disk build.

Comment (schaefi) [inline kiwi/builder/disk.py]: I just missed to add the key information... my bad, thanks for the review

**Pull request merged**